src/vertica_data.js

```diff
@@ -0,0 +1,99 @@
+var vertica = require('vertica');
+
+var GRAFANA_TABLE = "grafana_metric";
+var DROP_TABLE_QUERY = "DROP TABLE IF EXISTS " + GRAFANA_TABLE;
+var CREATE_TABLE_QUERY = "CREATE TABLE IF NOT EXISTS " + GRAFANA_TABLE + "(" +
+  "measurement VARCHAR(64), " +
+  "source VARCHAR(64), " +
+  "hostname VARCHAR(64), " +
+  "datacenter VARCHAR(64), " +
+  "createdAt TIMESTAMP, " +
+  "value FLOAT " +
+")";
+
+function live(program, config) {
+  console.log('connecting to ' + program.server + ' : ' + program.port);
+  var data = {};
+
+  var verticaConfig = {
+    host: program.server,
+    port: program.port,
+    user: config.user,
+    password: config.pwd,
+    database: config.db,
+    ssl: 'optional'
+  };
+
+  var connection = vertica.connect(verticaConfig);
+
+  connection.query(DROP_TABLE_QUERY, onDropTable);
+
+  function onDropTable(err, result) {
+    console.log(err, result);
+    if (!err) {
+      connection.query(CREATE_TABLE_QUERY, onCreateTable);
+    }
+  }
+
+  function onCreateTable(err, result) {
+    console.log(err, result);
+    if (!err) {
+      runRandomWalk();
+    }
+  }
+
+  function onInsertQuery(err, result) {
+    console.log(err, result);
```

Comment:
This doesn't add that much of information and would be nice to handle that better. console.error if err is defined, if not a console.log of "Metric written" or similar. Right now I get one of these at what seems to be each insert:
```
null Resultset {
  fields: 
   [ Field {
       name: 'OUTPUT',
       tableOID: 0,
       tableFieldIndex: 1,
       typeOID: 6,
       type: 'integer',
       size: 8,
       modifier: 8,
       formatCode: 0,
       decoder: [Function: integer] } ],
  rows: [ [ 1 ] ],
  notices: [],
  status: '' }
```